hack/histogram.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# This is a terrible ugly hack of a script.
+
+import re
+import sys
+import logging
+import argparse
+import termplotlib as tpl
+
+from codecs import decode
+from kubernetes import client, config
+from kubernetes.client import Configuration
+
+METRIC = 'etcd_request_duration_seconds'
+
+
+def main(type, *args, **kwargs):
+    global v1
+
+    try:
+        config.load_kube_config()
+        version = client.VersionApi().get_code()
+        logging.info(f"Connected to {Configuration._default.host} - {version.git_version}")
+    except Exception as e:
```

Comment:
Is there any specific behavior that might be associated with a particular exception we would want to catch or do we not care and just bail on any?

Reply:
This is essentially a baseline configuration + connection test. Since we're both loading the kubeconfig and making a request to the server to get the version, any basic error (bad config, can't connect to server, bad creds, etc) is handled here.